src/controllers/accountAdder/accountAdder.ts

```diff
@@ -573,14 +570,19 @@ export class AccountAdderController extends EventEmitter {
     }
 
     this.readyToAddAccounts = [
-      ...accounts.map((x) => ({
-        ...x.account,
-        newlyCreated: newlyCreatedAccounts.includes(x.account.addr)
-      }))
+      ...accounts.map(
+        (x, i) =>
+          ({
+            ...x.account,
+            preferences: getDefaultAccountPreferences(x.account.addr, this.#accounts.accounts, i),
+            newlyCreated: newlyCreatedAccounts.includes(x.account.addr)
+          } as Account & {
+            newlyCreated?: boolean | undefined
+          })
+      )
```

Comment:
Instead of `as Account & { newlyCreated?: boolean | undefined }`, this could be fixed if the `preferences` of the Account interface is optional, since it could be `undefined`, or not?